contracts/system-contracts/hedera-schedule-service/HederaScheduleService_HIP1215.sol

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.5.0 <0.9.0;
+
+import "../HederaResponseCodes.sol";
+import "./IHederaScheduleService_HIP1215.sol";
+
+abstract contract HederaScheduleService_HIP1215 {
```

Comment:
I think this contract does not apply

Only 1 HederaScheduleService should exist, with the new extensions inheritated